src/test/java/com/connexta/store/service/impl/StoreServiceImplTest.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.store.service.impl;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.emptyString;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.core.Is.is;
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.ArgumentMatchers.argThat;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+
+import com.connexta.store.adaptors.StorageAdaptor;
+import com.connexta.store.clients.IndexDatasetClient;
+import com.connexta.store.clients.TransformClient;
+import com.connexta.store.controllers.StoreController;
+import com.connexta.store.service.api.StoreService;
+import java.io.InputStream;
+import java.net.URI;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Ignore;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.MediaType;
+import org.springframework.web.util.UriComponentsBuilder;
+
+@ExtendWith(MockitoExtension.class)
+class StoreServiceImplTest {
+
+  @Test
+  @Ignore("TODO")
+  void testRetrieveFile() {}
```

Comment:
✏️ You can remove all the todos in this test class, I'm addressing them in my PR.